examples/Example01_Basic_OneShot/Example01_Basic_OneShot.ino

```diff
@@ -43,36 +45,35 @@ void setup() {
   Serial.println("Sensor began.");
 
   // Set measurement mode and change device operating mode to measure.
-  if(myUVSensor.startMeasurement(MEAS_MODE_CMD) == false) {
+  if(myUVSensor.prepareMeasurement(MEAS_MODE_CMD) == false) {
     Serial.println("Sensor did not get set properly.");
     Serial.println("Spinning...");
     while(1);
   }
 
   Serial.println("Set mode to command.");
-
 }
 
 void loop() {
   
   // Send a start measurement command.
-  if(SFE_BUS_OK != myUVSensor.setStartStateMode(START_STATE_ENABLED))
+  if(kSTkErrOk != myUVSensor.setStartState(true))
     Serial.println("Error starting reading!");
   
   // Wait for a bit longer than the conversion time.
-  delay(2+(1 << myUVSensor.getConfigConversionTime()));
+  delay(2+(1 << myUVSensor.getConversionTime()));
```

Comment:
I think it'd be good to add a helper method `getConversionTimeMillis()` (or whatever name) to help the user. It's not immediately obvious from this code why the bitshifting is needed, and some folks are going to assume `getConversionTime()` gives them milliseconds.

In fact, it might help to rename it to `getConversionTimeRaw()` to help further distinguish them.

Related - it might be good to do something similar with any methods that return enums corresponding to "real" values, including `getGain()`, `getCClk()`, and `getDigitalDividerRange()`. That is, add `Raw` to the end of those method names, and add helper methods for each: `getGainScaler()`, `getCClkMHz()`, and `getDigitalDividerRangeScaler()` (idk about `Scaler`, names are hard lol)

Reply:
We haven't really done this in any other library when you pull it out, but I definitely see the niceness of that. However, those functions aren't really used in the library at all, I just use the member variable directly. Think it's worth providing both raw and human-readable versions public to the user?

Reply:
> We haven't really done this in any other library

Actually, I've seen a mix within our libraries. Some return raw register values, some return "real" values ([example here](
https://github.com/sparkfun/SparkFun_MMC5983MA_Magnetometer_Arduino_Library/blob/8b22bc4e68e6f2331d419b0571dd93adc7a36f2f/src/SparkFun_MMC5983MA_Arduino_Library.cpp#L651)). It's not always clear when methods will return raw or "real" values, but in my mind, I sort of default to "real" values unless the method has `Raw` or something in the name.

> Think it's worth providing both...?

I'm guessing everyone's gonna have a different opinion on this, but I'd vote yes.

Reply:
I'm not opposed to providing both.